packages/NodeNameResolver/NodeNameResolver/ClassConstFetchNameResolver.php

```diff
@@ -5,25 +5,17 @@
 namespace Rector\NodeNameResolver\NodeNameResolver;
 
 use PhpParser\Node;
+use PhpParser\Node\Expr;
 use PhpParser\Node\Expr\ClassConstFetch;
+use PhpParser\Node\Expr\Error;
 use PHPStan\Analyser\Scope;
 use Rector\NodeNameResolver\Contract\NodeNameResolverInterface;
-use Rector\NodeNameResolver\NodeNameResolver;
-use Symfony\Contracts\Service\Attribute\Required;
 
 /**
  * @implements NodeNameResolverInterface<ClassConstFetch>
  */
 final class ClassConstFetchNameResolver implements NodeNameResolverInterface
 {
-    private NodeNameResolver $nodeNameResolver;
-
-    #[Required]
-    public function autowire(NodeNameResolver $nodeNameResolver): void
-    {
-        $this->nodeNameResolver = $nodeNameResolver;
-    }
-
```

Comment:
Awesome work :clap: 

If you can remove more `[#Required]` parts, the better! The less cyclic design, the better :)